### PR TITLE
fix typos

### PR DESCRIPTION
**Pull request by Jojo-Schmitz**

### CODE REVIEW

File: mscore/importmidi/importmidi_model.cpp [hunk 1]
```diff
@@ -505,7 +505,7 @@ void TracksModel::reset(const MidiOperations::Opers &opers,
                         {
                         }
                   QString headerName() const override { return QCoreApplication::translate(
-                                          "MIDI import operations", "2x less\nmeasure count"); }
+                                          "MIDI import operations", "Halving the\nmeasure count"); }
```
Comment (Howard-C) [inline mscore/importmidi/importmidi_model.cpp:508]: Then the `\n` won't be necessary I guess?

Comment (Jojo-Schmitz) [reply mscore/importmidi/importmidi_model.cpp:508]: It was needed for the shorter string before, so surely is still needed

File: avsomr/ret.cpp [hunk 2]
```diff
@@ -151,7 +151,7 @@ QString Ret::supportHint(Code c)
             case NetworkError:      return QObject::tr("Check your internet connection");
             case ServerError:       return PleaseTryAgain;
                   // local
-            case LocalNotInstalled: return QObject::tr("Try to uncheck and check again use the local OMR engine");
+            case LocalNotInstalled: return QObject::tr("Try to uncheck and check again to use the local OMR engine");
```
Comment (igorkorsukov) [inline avsomr/ret.cpp:154]: Thank you for your correction.

File: share/instruments/instruments.xml [hunk 3]
```diff
@@ -10751,9 +10751,9 @@
                    <genre>popular</genre>
              </Instrument>
              <Instrument id="laud">
-                   <longName>Laud</longName>
-                   <shortName>Laud</shortName>
-                   <description>Spanish Laud</description>
+                   <longName>Laúd</longName>
+                   <shortName>Laúd</shortName>
+                   <description>Spanish Laúd</description>
```
Comment (Jojo-Schmitz) [inline share/instruments/instruments.xml:10756]: See https://en.wikipedia.org/wiki/La%C3%BAd

**Pull request merged**